pkcs11/_pkcs11.pyx

```diff
@@ -1355,6 +1355,11 @@ cdef class lib:
         except StopIteration:
             return token
 
+    def reinitialize(self):
```

Comment:
Can you please add a docstring? It will also need one in the *.py file that will be generated into the API docs.

Reply:
FYI docstrings go in `types.py`

Reply:
Perhaps you can help me understand, but there is no class in types.py that the `cdef class Lib` shadows. How would you like me to proceed?

Reply:
Ah you're right, `lib` is magic and so wouldn't auto-inspect. It's manually documented in `api.rst` from line 15 onwards.